Swift/Tests/SampleCodeTest.swift

```diff
@@ -496,6 +496,30 @@ class SampleCodeTest: CBLTestCase {
         // # end::replication-logging[]
     }
 
+    func dontTestReplicationBasicAuthentication() throws {
+        // # tag::basic-authentication[]
+        let url = URL(string: "ws://localhost:4984/mydatabase")!
+        let target = URLEndpoint(url: url)
+        let config = ReplicatorConfiguration(database: database, target: target)
+        config.authenticator = BasicAuthenticator(username: "john", password: "pass")
+
+        self.replicator = Replicator(config: config)
+        self.replicator.start()
+        // # end::basic-authentication[]
+    }
+
+    func dontTestReplicationSessionAuthentication() throws {
```

Comment:
We have SessionAuthenticator for this. Can you change to use that?

Reply:
Ok - I'm a bit confused reading comments in https://issues.couchbase.com/browse/CBSE-5104

When should we recommended to use SessionAuthenticator vs. setting the cookie in the header?

Reply:
For SG Session, we should recommend to use SessionAuthenticator. The customer header is for something in addition e.g. proxy required headers or cookies.